visualization/app/codeCharta/ui/mapTreeView/mapTreeView.level.component.ts

```diff
@@ -47,12 +47,7 @@ export class MapTreeViewLevelController implements BuildingHoveredSubscriber, Bu
 	}
 
 	public onBuildingHovered(hoveredBuilding: CodeMapBuilding) {
-		this._viewModel.isHoveredInCodeMap = !!(
-			this.node &&
-			this.node.path &&
-			hoveredBuilding.node &&
-			hoveredBuilding.node.path === this.node.path
-		)
+		this._viewModel.isHoveredInCodeMap = Boolean(this.node?.path && hoveredBuilding.node?.path === this.node.path)
```

Comment:
Is the cast even necessary? This evaluates to a boolean anyway

Reply:
It's not strictly necessary but the return type is `boolean`. That is only guaranteed in case of the cast. Otherwise it might be `null`, `undefined` or an empty string.